src/scripts/data-node-install.sh

```diff
@@ -94,7 +94,7 @@ COUNTRY=""
 INSTALL_SWITCHES=""
 
 #Loop through options passed
-while getopts :n:m:v:A:R:K:S:Z:p:U:I:c:e:f:g:t:s:o:a:k:L:C:B:Xxyzldjh optname; do
+while getopts :n:m:v:A:R:K:S:Z:p:U:I:c:e:f:g:t:s:o:a:k:L:C:B:E:Xxyzldjh optname; do
```

Comment:
Want to keep the provisioning of a storage account for azure-repository plugin separate.